src/series_sum.py

```diff
@@ -0,0 +1,8 @@
+def series_sum(n):
+    series = [1]
+    y = 1
+    for num in range(n):
+        series.append(1 / (y + 3))
+        y = y + 3
+    ans = sum(series[:n])
+    return str('%.2f' % ans)
```

Comment:
You shouldn't need to wrap `'%.2f' % ans` in `str` since it's already a formatted string.